tasks.php

```diff
@@ -114,49 +114,49 @@
     6 => "Trivial",
     7 => "Enhancement",
 ];
-$priority_array = [
+const TASK_PRIORITY = [
     1 => "Very High",
     2 => "High",
     3 => "Medium",
     4 => "Low",
     5 => "Very Low",
 ];
-$categories_array = [
-    1 => "None",
+const TASK_CATEGORIES = [
+    7 => "Activity Hub",
     2 => "Documentation",
     3 => "Entrance",
+    32 => "Format Preview",
+    21 => "Forums, Private Messages",
+    20 => "HTML Pool",
     4 => "Log in/out",
+    24 => "Mentoring",
+    25 => "My Projects",
     5 => "New Member",
-    6 => "Proofreading Interface",
-    7 => "Activity Hub",
+    1 => "None",
+    19 => "OCR Pool",
+    99 => "Other",
+    26 => "Page Details, Diffs",
     8 => "Post-Processing",
-    9 => "Preferences",
     10 => "Pre-Processing",
-    11 => "Project Page",
+    9 => "Preferences",
     12 => "Project Lists",
     13 => "Project Manager",
+    31 => "Project Notifications",
+    11 => "Project Page",
+    23 => "Project Search",
+    6 => "Proofreading Interface",
+    29 => "Release Queues",
+    28 => "Rounds",
     14 => "Site wide",
+    18 => "Smooth Reading",
     15 => "Statistics, Page Counts",
-    16 => "Translation",
     17 => "Task Center",
-    18 => "Smooth Reading",
-    19 => "OCR Pool",
-    20 => "HTML Pool",
-    21 => "Forums, Private Messages",
+    16 => "Translation",
     22 => "WordCheck",
-    23 => "Project Search",
-    24 => "Mentoring",
-    25 => "My Projects",
-    26 => "Page Details, Diffs",
     27 => "Quizzes",
-    28 => "Rounds",
-    29 => "Release Queues",
     30 => "Teams",
-    31 => "Project Notifications",
-    32 => "Format Preview",
-    99 => "Other",
+    // 32 max
```

Comment:
Not sure what the comment is for -- there are actually 33 with `"99 => Other"`

Reply:
This is to help the developer know which is the "highest that's not 99" key. I grok what Brian is doing here.